rust/src/shards/gui/containers/area.rs

```diff
@@ -6,6 +6,7 @@ use super::Area;
 use crate::shard::Shard;
 use crate::shards::gui::containers::ANCHOR_TYPES;
 use crate::shards::gui::util;
+use crate::shards::gui::util::get_current_context;
```

Comment:
Just a personal preference, but I like to keep the namespace for those helper functions. Same as with `util::activate_ui_contents(context, input, ui...)` we could have `util::get_current_context(&self.instance)`.